### PR TITLE
service level objectives

### PR DESCRIPTION
**Pull request by DanielMcSheehy**
Screen Shot 2021-03-15 at 12 14 26 PM
Screen Shot 2021-03-15 at 12 14 36 PM
Screen Shot 2021-03-15 at 12 14 43 PM
Screen Shot 2021-03-16 at 2 39 59 PM

### CODE REVIEW

File: product-releases/2021-03-12-service-level-objectives.mdx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Service Level Objectives
+tags: [service level objectives, standards]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<div class="avatar margin-vert--md minus-margin-top">
```
Comment (weeyum) [inline product-releases/2021-03-12-service-level-objectives.mdx:8]: this is not what we usually do for avatars, can you copy the ones from other blog posts (not the standards one)

Comment (DanielMcSheehy) [reply product-releases/2021-03-12-service-level-objectives.mdx]: good catch, sure

File: product-releases/2021-03-15-service-level-objectives.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  src={useBaseUrl('img/service-level-objectives/standard-slo-criteria.png')}
+/>
+
+In addition, standards now allow you to observe which services are missing key service level objectives, which displays
+obvious indicators on where improvements can be made.
+
+<img
+  alt='SLO Standards'
+  src={useBaseUrl('img/service-level-objectives/service-slo-criteria.png')}
+/>
```
Comment (weeyum) [inline product-releases/2021-03-15-service-level-objectives.mdx:45]: below, can we remove the `.DS_Store` files? maybe we should add it to gitignore

**Review by weeyum — APPROVED** [review]
approved, but perhaps we can let @Ttsunekawa and @elliejy merge when we launch it to all customers? are we ready to do that?

order:
open feature flag to everyone
merge release post
deploy announcement banner addition

File: product-releases/2021-03-15-service-level-objectives.mdx (hunk 2 above)
Comment (DanielMcSheehy) [reply product-releases/2021-03-15-service-level-objectives.mdx:45]: good call

Comment (weeyum) [reply product-releases/2021-03-15-service-level-objectives.mdx:45]: @DanielMcSheehy there's still 2 more ds store files i think

**Review by Ttsunekawa — CHANGES_REQUESTED** [review]
can we add a step in here about tagging their SLOs with service name?

File: product-releases/2021-03-15-service-level-objectives.mdx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+<br />
+
+:::note
+For each service level objective, please tag (`service:<service_name>`) 
+the service that is associated with it.
+<img
+  alt='Datadog Service Tagging'
+  src={useBaseUrl('img/service-level-objectives/datadog-tag.png')}
+/>
+:::
```
Comment (Ttsunekawa) [inline product-releases/2021-03-15-service-level-objectives.mdx:57]: cool TIL :::note:::

**Review by Ttsunekawa — APPROVED** [review]

**Comment by Ttsunekawa** [comment]
thanks Daniel!

**Pull request merged**